src/ProposalPayload.sol

```diff
@@ -1,17 +1,23 @@
 // SPDX-License-Identifier: AGPL-3.0-only
-pragma solidity ^0.8.15;
+pragma solidity 0.8.17;
 
-import {AaveMisc} from "@aave-address-book/AaveMisc.sol";
-import {AaveV2Ethereum} from "@aave-address-book/AaveV2Ethereum.sol";
+import {AaveV2Ethereum, AaveV2EthereumAssets} from "aave-address-book/AaveV2Ethereum.sol";
 
 /**
- * @title <TITLE>
+ * @title CRV Interest Rate Curve Upgrade
  * @author Llama
- * @notice <DESCRIPTION>
- * Governance Forum Post:
- * Snapshot:
+ * @notice Amend CRV interest rate parameters on the Aave Ethereum v2liquidity pool.
+ * Governance Forum Post: https://governance.aave.com/t/arfc-crv-interest-rate-curve-upgrade/11337
+ * Snapshot: https://snapshot.org/#/aave.eth/proposal/0x56aaf192f5cad8277b0e7c82abad030c62bb8fcfe4f2640ce5b896ab04397c20
  */
 contract ProposalPayload {
+    address public constant INTEREST_RATE_STRATEGY = 0x005cb4DcEC37feE53CB4330A3A59e8B9F3dDC86e;
+
     /// @notice The AAVE governance executor calls this function to implement the proposal.
```

Comment:
this comment is a bit weird tbh (as it's wrong - the short executor executes this via delegatecall, not the governance), would just remove it though (same in the other proposal, where the bridgeexecutor is the one)